### PR TITLE
PT-164050506 Bump minimum gasprice

### PR DESCRIPTION
**Pull request by hanssv**

### CODE REVIEW

File: data/aecore/.genesis/accounts_test.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-   "ak_2a1j2Mk9YSmC1gioUq4PWRm3bsv887MbuRVwyv4KaUGoR1eiKi": 100000000000001
+   "ak_2a1j2Mk9YSmC1gioUq4PWRm3bsv887MbuRVwyv4KaUGoR1eiKi": 100000000000000000000001
```
Comment (lucafavatella) [inline data/aecore/.genesis/accounts_test.json]: In this version of the code i.e. without 9efcab30 this impacts testnet too.
This PR may have some not explicit conflicts with 9efcab30

File: system_test/only_local/aest_heavy_sync_SUITE_data/aeternity.yaml.mustache [hunk 2]
```diff
@@ -1,40 +0,0 @@
-{{#aeternity_config}}
```
Comment (lucafavatella) [inline system_test/only_local/aest_heavy_sync_SUITE_data/aeternity.yaml.mustache]: ???

File: data/aecore/.genesis/accounts_test.json (hunk 1 above)
Comment (lucafavatella) [reply data/aecore/.genesis/accounts_test.json]: I see the rebase included the commit that copied this file with other name for testnet, so this is resolved.

**Comment by lucafavatella** [comment]
I see this includes (attempted) fix for running heavy sync system test (file relocation). Good.

File: apps/aechannel/src/aesc_fsm.erl [hunk 3]
```diff
@@ -1918,9 +1918,13 @@ pay_close_mutual_fee(Fee, IAmt, RAmt) ->
...
     end.
 
+-ifdef(TEST).
 close_mutual_defaults(_Account, _D) ->
-    #{ fee   => 20000}.
-
+    #{ fee   => 20000 }.
+-else.
+close_mutual_defaults(_Account, _D) ->
+    #{ fee   => 20000 * 1000000000 }.
```
Comment (lucafavatella) [inline apps/aechannel/src/aesc_fsm.erl]: In this same file I see `default_fee(_Tx) -> 20000.` used in multiple places. I have not got my head around that yet. Hence:
* Not sure if any points using default_fee require amendments.
* I see that this close_mutual_defaults is used as first element in `maps:merge(Def, Opts)` - not sure if any Opts has fee hence overwriting this. (And making this changed line ineffective.)

Comment (lucafavatella) [inline apps/aechannel/src/aesc_fsm.erl]:
```suggestion
    #{ fee   => 20000 * 1000000 }.
```

File: docs/release-notes/next-minerva/PT-164050506-bump_minimum_gasprice.md [hunk 4]
```diff
@@ -0,0 +1 @@
+* Set the minimum gas price to 1000000000 in order to make transactions more reasonably priced.
```
Comment (lucafavatella) [inline docs/release-notes/next-minerva/PT-164050506-bump_minimum_gasprice.md]:
```suggestion
* Set the minimum gas price to 1000000 in order to make transactions more reasonably priced.
```

File: system_test/helpers/aest_nodes.erl [hunk 5]
```diff
@@ -169,6 +171,10 @@
 
 -export_type([test_ctx/0]).
 
+%% Use this gas price in tests
+gas_price() ->
+    1000000000.
```
Comment (lucafavatella) [inline system_test/helpers/aest_nodes.erl]:
```suggestion
    1000000.
```

File: apps/aechannel/src/aesc_fsm.erl (hunk 3 above)
Comment (hanssv) [reply apps/aechannel/src/aesc_fsm.erl]: Why, I would just have to change it again in the miner config PR?

File: system_test/helpers/aest_nodes.erl (hunk 5 above)
Comment (hanssv) [reply system_test/helpers/aest_nodes.erl]: No. Letting the system test use realistic values is a good idea.

File: apps/aechannel/src/aesc_fsm.erl (hunk 3 above)
Comment (hanssv) [reply apps/aechannel/src/aesc_fsm.erl]: I trust Ulf to solve this properly.

File: apps/aecore/src/aec_governance.erl [hunk 6]
```diff
@@ -108,8 +109,16 @@ tx_base_gas(oracle_response_tx) -> ?TX_BASE_GAS.
 byte_gas() ->
     ?BYTE_GAS.
 
-minimum_gas_price() ->
+-ifdef(TEST).
```
Comment (lucafavatella) [inline apps/aecore/src/aec_governance.erl]: Ok though we should consider remoiving this.
[Tracked](https://www.pivotaltracker.com/story/show/164103197).

File: apps/aechannel/src/aesc_fsm.erl (hunk 3 above)
Comment (lucafavatella) [reply apps/aechannel/src/aesc_fsm.erl]: This is [tracked](https://www.pivotaltracker.com/story/show/164072144).

**Comment by lucafavatella** [comment]
I see only missing job is [Linux tests on appveyor](https://ci.appveyor.com/project/aeternity/aeternity/builds/22508916/job/wvi71rkgtuuk08ff). That kind of job was already run on Circle CI.

**Comment by lucafavatella** [comment]
All CI passed.

**Pull request merged**